conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -521,8 +521,14 @@ public void testEnumHeaderParameter() {
     public void testListOfNull() {
         assertThatThrownBy(() ->
                         client.receiveListOfStrings(AuthHeader.valueOf("authHeader"), Collections.singletonList(null)))
-                .isInstanceOfSatisfying(
-                        RemoteException.class, re -> assertThat(re.getStatus()).isEqualTo(400));
+                .isInstanceOfSatisfying(RemoteException.class, re -> {
+                    assertThat(re.getStatus())
+                            // Jackson < 2.17 throws `NullPointerException` resulting in HTTP 400.
+                            // Jackson > 2.17 throws "MismatchedInputException: Guava `Collection` of type
+                            // `com.google.common.collect.ImmutableList<java.lang.String>` does not accept
+                            // `null` values" resulting in HTTP 422.
```

Comment:
This comment isn't really relevant - after upgrading we're not longer using Jackson < 2.17.

Reply:
removed